test/tap/pack-file-inclusion.js

```diff
@@ -0,0 +1,82 @@
+// verify that pack includes only the files/dirs we want
+var test = require('tap').test
+var common = require('../common-tap')
+var fs = require('graceful-fs')
+var join = require('path').join
+var mkdirp = require('mkdirp')
+var rimraf = require('rimraf')
+var tar = require('tar')
+var zlib = require('zlib')
+
+var pkg = join(__dirname, 'pack-age')
```

Comment:
I prefer to name these after the test– if the test crashes, it'll be clear which one made it, plus it ensures tests don't stomp on each other. (Relatedly, someday I'd like to be able to run tests concurrently. Currently tests often setup stuff listening on fixed port #s, so they can't, but…)